source/documentation/contribute-to-rdoproject.html.md

```diff
@@ -0,0 +1,92 @@
+---
+author: chandankumar
+title: Let's Contribute to RDO Project
+---
```

Comment:
I think the content of this document could directly go to the onboarding guide in https://www.rdoproject.org/documentation/onboarding/. My proposal would be to have an onboarding guide with a brief list of ordered tasks (more or less what we have now) first, and below detailed howtos linked from short descriptions, what do you think?